crates/cli/src/subcommands/logout.rs

```diff
@@ -0,0 +1,12 @@
+use crate::Config;
+use clap::{ArgMatches, Command};
+
+pub fn cli() -> Command {
+    Command::new("logout")
+}
+
+pub async fn exec(mut config: Config, _args: &ArgMatches) -> Result<(), anyhow::Error> {
```

Comment:
You also have to do `POST /auth/cli/logout` with same Authorizaton headers as sent on `/api/spacetimedb-token`, to delete this session from auth service also

Reply:
It also should take `auth-type` flag for that I assume

Reply:
@PuruVJ what's the `auth-type` flag? What would I pass?

Reply:
Oh sorry I meant auth-host 😅

Reply:
ah okay :+1: in that case done